Automattic/wp-calypso
# BBE: Revamp new/existing site step

Fixes https://github.com/Automattic/martech/issues/2327

## Proposed Changes

* Updates the new/existing site step of the BBE flow with the <DIFMLanding/> component.

<img width="1481" alt="image" src="https://github.com/Automattic/wp-calypso/assets/5436027/fb1e6cdd-745b-4cfd-ad98-567bee0d4f79">



## Testing Instructions



* Go to `/start/do-it-for-me`.
* Confirm that the page matches the screenshot.
* Confirm both the new site and existing site flows work correctly.

-----

* Go to `/start/do-it-for-me` in an incognito window and create a new user.
* Confirm that this step is shown. In production, we skip this step for new users.

-----

* Go to `/setup/site-setup/difmStartingPoint?siteSlug=<site slug>` and confirm that this page is unchanged from production.

## Pre-merge Checklist
